vectordraw.py

```diff
@@ -67,9 +120,25 @@ def check_angle(check, vectors):
     y = vec.tip.y - vec.tail.y
     dot_product = x * math.cos(expected) + y * math.sin(expected)
     angle = math.degrees(math.acos(dot_product / vec.length))
-    if abs(angle) > tolerance:
+    return abs(angle) > tolerance
```

Comment:
@mtyaka nit: same here - I find it counter-intuitive that this function returns False if the check pass.